### PR TITLE
Option to encrypt saves with a password

### PR DESCRIPTION
**Pull request by ShnitzelKiller**
With this feature, you can type /encrypt, which prompts you with a password. That password will be used to encrypt/decrypt any saves in that session. When loading a game at the start, you will also be prompted with a password if the save is encrypted.

### CODE REVIEW

**Comment by ShnitzelKiller** [comment]
Note that this PR contains #30.

**Comment by thadunge2** [comment]
How would you enter the password if you're loading from the menu?

**Comment by ShnitzelKiller** [comment]
> 
> 
> How would you enter the password if you're loading from the menu?

It detects if the file cannot be loaded normally, and prompts you for a password.

```
0) New Game
1) Load Game

Enter the number of your choice: 1
What is the ID of the saved game? (prefix with gs:// if it is a cloud save) be5fee78-22d5-11ea-af86-0103c3fee881

Loading Game...

Enter password (if this is an encrypted save):
encryption set (disable with /encrypt)
You are Lariesa, a wizard from the kingdom of Thema. You have a amulet and a crystal ball.
```

**Comment by thadunge2** [comment]
Fair enough! Looks like there's a conflict now, though. Oh, and you need to add your cryptography modules to requiresments.txt.

**Comment by ShnitzelKiller** [comment]
I fixed the conflict. Also I thought I did add it to requirements.txt.
EDIT: Whoops wrong button lol. But yeah there are multiple requirements.txt, and I put it in the top level one.

File: requirements.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ tensorflow==1.15
 func_timeout
 playsound
 tracery
+cryptography
```
Comment (ShnitzelKiller) [inline requirements.txt:11]: added requirement

**Pull request merged**